WalletWasabi.Bench/WalletWasabi.Bench.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is that intentional that you did not add this project to the solution?

Reply:
I have added it to the solution.